### ISSUE TITLE
Enable parsing jsx files

### ISSUE THREAD

**Issue by kutnickclose** [43081j/postcss-lit#30]
Hello,

I am using postcss-lit as a custom syntax for stylelint but it bombs out trying to parse my jsx/tsx files. Could jsx be added to the babel parser? 

```
// parse.ts
const ast = babelParse(sourceAsString, {
    sourceType: 'unambiguous',
    plugins: [
      'typescript',
      ['decorators', {decoratorsBeforeExport: true}],
+    'jsx'
    ],
    ranges: true
  });
```

Thanks,

Tim

**Comment by kutnickclose** [comment]
Thanks!

**Comment by kutnickclose** [comment]
Would it be possible to publish a new version?

**Comment by 43081j** [comment]
i have another bug i need to look at but after that, yes, ill publish a new version :+1:

**Comment by kutnickclose** [comment]
FYI,

To get stylelint plugin to work with jsx/tsx you'll need `"javascriptreact", 'typescriptreact"` in the stylelint.validate.
```
{
  "stylelint.validate": ["css", "javascript", "typescript", "javascriptreact", 'typescriptreact"]
}
```

**Comment by 43081j** [comment]
fyi have published 0.5.0

let me know if you run into any problems
